packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -49,9 +49,13 @@ export async function errorHandling(errorDetails: any, label?: string, moreInfo?
         // open config file for missing hostname error
         const msg = errorDetails.toString();
         if (msg.includes("hostname")) {
-            if ((await globals.PROFILESCACHE.getProfileInfo()).usingTeamConfig) {
+            let mProfileInfo = await globals.PROFILESCACHE.getProfileInfo();
+            if (!mProfileInfo) {
+                mProfileInfo = await Profiles.getInstance().getProfileInfo();
+            }
```

Comment:
I'm curious if there is a better way to implement this.

Maybe modifying the `getProfileInfo` to return the instance stored in the class if defined
If not, then create a new one ? 
(This may have other implications though)

I guess I can see myself forgetting that the `globals` instance can be `undefined`.

Reply:
Hey Fernando, I know we spoke of possibly creating an instance in the future of getProfileInfo() to improve performance.  I wonder if we need to create a new story to handle that since here with the globals it is actually an instance of ProfilesCache that is created during activation.  Not sure why the call is failing in this case but not others though.